libmscore/score.h

```diff
@@ -997,7 +1000,7 @@ class Score : public QObject, public ScoreElement {
       void removeAudio();
 
       void doLayout();
-      void doLayoutRange(int, int);
+      void doLayoutRange(int stick, int etick);
```

Comment:
parameter names not needed

Reply:
well I realize that...to me including the parameters is nicer, but I'll remove that from now on.

Reply:
In this particular case I would keep the names. The reason is simple, when you want to use this or that method, you look at the declaration, not the definition. Two int parameters may mean everything. But stick, etick make things clear to the reader.

You don't need to specify names if parameter type is obvious (e.g. Score*), but in this case I strongly recommend to keep the names.